README.md

```diff
@@ -32,66 +32,66 @@ Current build status
         <table>
           <thead><tr><th>Variant</th><th>Status</th></tr></thead>
           <tbody><tr>
-              <td>linux_python3.6</td>
+              <td>linux_python3.6.____cpython</td>
```

Comment:
@conda-forge-admin , why all the `.____cpython` appending? Is this intentional? Please clarify. Thanks!

Reply:
New changes for shipping pypy. All of these things are auto generated. No need to worry. Merge away!

Reply:
Great. Thanks for the quick reply!